### ISSUE TITLE
Improve accessibility (a11y) particularly the colour contrast

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/genesis-tags#8]
I had a discussion with my old boss from IBM today about accessibility and WordPress. 
He's now an Ambassador for the Digital Poverty Alliance and wanted to find out more about WordPress's commitment to a11y.
We looked at some websites which had problems with green on white or vice versa.
Looks like we should be updating the theme for TAGS.

### Recommendations
- Test using WAVE ( web accessibility evaluation tool ), which is powered by WebAIM
- Fix where possible

eg
- Make the greens darker

**Comment by bobbingwide** [comment]
On 30 Nov and 7 Dec, the AIM Score for oik-plugins.com was 2.1 - 2.3

On `s.b/tags`, with change to `custom.css` and `style.css`, made on 5th May,  the WebAIM score is now 9.9 
There's 1 Error and 5 Alerts
